packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/InternalTypedRealm.kt

```diff
@@ -36,7 +36,7 @@ internal interface InternalTypedRealm : TypedRealm {
 
     override fun <T : BaseRealmObject> query(clazz: KClass<T>, query: String, vararg args: Any?): RealmQuery<T> {
         val className = configuration.mediator.companionOf(clazz).`io_realm_kotlin_className`
-        return ObjectQuery(realmReference, realmReference.schemaMetadata.getOrThrow(className).classKey, clazz, configuration.mediator, null, query, *args)
+        return ObjectQuery(realmReference, realmReference.schemaMetadata.getOrThrow(className).classKey, clazz, configuration.mediator, query, args)
```

Comment:
Nice change...having to remember the vararg across the entire internal API was pretty errorprone